src/mount/apple.rs

```diff
@@ -0,0 +1,125 @@
+use crate::{Errno, NixPath, Result};
+use libc::c_int;
+
+libc_bitflags!(
+    /// Used with [`Nmount::nmount`].
+    pub struct MntFlags: c_int {
+        /// Do not interpret special files on the filesystem.
+        MNT_NODEV;
+        /// file system supports content protection
+        MNT_CPROTECT;
+        /// filesystem is stored locally
+        MNT_QUARANTINE;
+        /// filesystem is stored locally
+        MNT_LOCAL;
+        /// quotas are enabled on filesystem
+        MNT_QUOTA;
+        /// identifies the root filesystem
+        MNT_ROOTFS;
+        /// FS supports volfs (deprecated flag in Mac OS X 10.5)
+        MNT_DOVOLFS;
+        /// file system is not appropriate path to user data
+        MNT_DONTBROWSE;
+        /// VFS will ignore ownership information on filesystem objects
+        MNT_IGNORE_OWNERSHIP;
+        /// filesystem was mounted by automounter
+        MNT_AUTOMOUNTED;
+        /// filesystem is journaled
+        MNT_JOURNALED;
+        /// Don't allow user extended attributes 
+        MNT_NOUSERXATTR;
+        /// filesystem should defer writes 
+        MNT_DEFWRITE;
+        /// don't block unmount if not responding
+        MNT_NOBLOCK;
+        /// file system is exported
+        MNT_EXPORTED;
+        /// All I/O to the file system should be done asynchronously.
+        MNT_ASYNC;
+        /// Force a read-write mount even if the file system appears to be
+        /// unclean.
```

Comment:
No idea what this flag is:

1. Kernel code says: force unmount or readonly change
2. Man page says: the filesystem should be forcibly unmounted even if files are still active

Reply:
Yeah, MacOS has some gap doc sometimes, I just follow BSD doc in this flag. Should we need to change?

Reply:
> Should we need to change?

We cannot do things right if Apple doesn't take their things seriously, so I guess no